internal/core/operations/healthcontroller/health_controller_executor.go

```diff
@@ -167,7 +165,7 @@ func (ex *SchedulerHealthControllerExecutor) ensureDesiredAmountOfInstances(ctx
 	switch {
 	case actualAmount > desiredAmount: // Need to scale down
 		removeAmount := actualAmount - desiredAmount
-		removeOperation, err := ex.operationManager.CreatePriorityOperation(ctx, op.SchedulerName, &remove_rooms.RemoveRoomsDefinition{
+		removeOperation, err := ex.operationManager.CreatePriorityOperation(ctx, op.SchedulerName, &remove.RemoveRoomsDefinition{
```

Comment:
What do you think to simplify the name of this struct to be only `Definition`  so the client will call `remove.Definition`, `add.Definition` it will be simpler.